test/stellar.test.ts

```diff
@@ -0,0 +1,64 @@
+import StellarSdk, { Keypair, Memo, MemoText } from "stellar-sdk";
+import axios from "axios";
+import {
+  AccountKeypair,
+  SigningKeypair,
+} from "../src/walletSdk/Horizon/Account";
+
+import sdk from "../src";
+const { walletSdk } = sdk;
+
+describe("Stellar", () => {
+  it("should create and submit a transaction", async () => {
+    const wal = walletSdk.Wallet.TestNet();
+    const stellar = wal.stellar();
+
+    // make sure signing key exists
+    const kp = SigningKeypair.fromSecret(
+      "SAS372GXRG6U7FW6W2PRVELKPOJG2FZZUADCIELWU2U3A45TNWXEQUV5"
+    );
+    try {
+      await stellar.server.loadAccount(kp.publicKey);
+    } catch (e) {
+      await axios.get("https://friendbot.stellar.org/?addr=" + kp.publicKey);
+    }
+
+    const now = Math.floor(Date.now() / 1000) - 5;
+
+    const txBuilderParams = [
+      {
+        sourceAddress: kp,
+        baseFee: 100,
+        startingBalance: 2,
+      },
+      {
+        sourceAddress: kp,
+        baseFee: 100,
+        timebounds: 180,
+        memo: new Memo(MemoText, "test-memo"),
+        startingBalance: 2.5,
+      },
+      {
+        sourceAddress: kp,
+        baseFee: 100,
+        timebounds: { minTime: now, maxTime: now + 180 },
+        startingBalance: 100,
+      },
+    ];
+
+    for (const param of txBuilderParams) {
```

Comment:
Can you also check that new account was indeed created?

Reply:
👍https://github.com/stellar/typescript-wallet-sdk/pull/26/commits/e127fbdea05e2209c13d6ca7920b776c5b0e9be4